macos/app_settings.podspec

```
@@ -0,0 +1,20 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint app_settings.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'app_settings'
+  s.version          = '5.1.1'
+  s.summary          = 'A Flutter plugin for opening macOS desktop settings from an app.'
+  s.description      = 'A Flutter plugin for opening macOS desktop settings from an app.'
+  s.homepage         = 'https://github.com/spencerccf/app_settings'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'ask.melnyk@gmail.com' => 'ask.melnyk@gmail.com' }
```

This should refer to the author/publisher of the plugin, not your own email address.

See https://github.com/spencerccf/app_settings/blob/master/ios/app_settings.podspec#L12

```suggestion
  s.author           = { 'spencerccf@gmail.com' => 'spencerccf@gmail.com' }
```